cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -4411,6 +4411,19 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    ig.push(
+        Inst::new(
+            "fcvt_low_from_uint",
+            r#"
+        Converts packed unsigned doubleword integers to packed double precision floating point.
```

Comment:
Please, use `32-bit integers` instead of `doubleword integers`.

You should also update the `fcvt_low_from_sint` description.

Reply:
@akirilov-arm good catch. Thanks!

Reply:
Yes, exactly, the term "doubleword" means "64-bit" in the Arm architecture.